src/main.cpp

```diff
@@ -839,10 +837,6 @@ static bool AcceptToMemoryPoolWorker(CTxMemPool &pool,
 
         // Set extraFlags as a set of flags that needs to be activated.
         uint32_t extraFlags = 0;
-        if (hasMay152018)
```

Comment:
@gandrewstone told me to use a `CTweak` just for CDS/V - but wouldn't it make the most sense to add a SCRIPT_ENABLE_NOV2018_OPCODES flag and use the code here in the same way as for the may upgrade?

Reply:
I think @gandrewstone told you to that while waiting for this PR to be finalized. I think that you can use the `IsNov152018Enabled()` helper and do it the same way we did in May.

Reply:
Makes sense, thanks.